Cargo.toml

```diff
@@ -27,6 +27,7 @@ indexmap = "2.1.0"
 itertools = "0.11.0"
 logos = "0.13.0"
 shrinkwraprs = "0.3.0"
+highlight_error = "0.1.0"
```

Comment:
Why did you specify not the latest version? I know that it doesn't matter for the patch number, but I just think it's strange to do.